openarm_hardware/src/canbus.cpp

```diff
@@ -31,7 +41,34 @@ CANBus::~CANBus() {
     close(sock_);
 }
 
+int CANBus::which_CAN(){
```

Comment:
It seems that this project uses `camelCase` not `camel_Case`:

```suggestion
int CANBus::whichCAN(){
```